doc/website/advanced/installation-guide-for-contributors.md

```diff
@@ -2,95 +2,111 @@
-- [LeakSanitizer](https://clang.llvm.org/docs/LeakSanitizer.html)
-LeakSanitizer is a run-time memory leak detector. In iceoryx , it runs as part of the AdderssSanitizer.
-- [UndefinedBehaviorSanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html)
-UndefinedBehaviorSanitizer (UBSan) is a fast undefined behavior detector. Iceoryx uses default behaviour ie `print a verbose error report and continue execution`
+The below-listed sanitizers are enabled at the moment.
+
+- [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) (ASan) is a fast memory error detector.
+!!! note
+    AddressSanitizer exits on the first detected error, which means there could be more errors in the codebase when this error is reported.
+- [LeakSanitizer](https://clang.llvm.org/docs/LeakSanitizer.html) (LSan) is a run-time memory leak detector. In iceoryx, it runs as part of the AddressSanitizer.
+- [UndefinedBehaviorSanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html) (UBSan) is a fast undefined behavior detector. iceoryx uses default behavior i.e. `print a verbose error report and continue execution`
+
+With the `iceoryx_build_test.sh` script you can do the scan on your own. Additionally, the scans are running on the CI in every Pull-Request.
+As a prerequisite, you need to install the clang compiler:
 
-In iceoryx are scripts available to do the scan on your own. Additionally the Scans are running on the CI in every Pull-Request.
-As Requirement you should install the clang compiler:
 ```bash
 sudo apt install clang
 ```
 
-Then you need to compile the iceoryx with the sanitizer flags:
+Then you need to compile iceoryx with the sanitizer flags:
+
 ```bash
 ./tools/iceoryx_build_test.sh build-strict build-all sanitize clang clean
 ```
-After that we can run the tests with enabled sanitizer options:
+
+Now we can run the tests with enabled sanitizer options:
+
 ```bash
-cd build
-../tools/run_tests.sh
+cd build && ./tools/run_tests.sh
 ```
-When the tests are running without errors then it is fine, else an error report is shown with a stacktrace to find the place where the leak occurs. If the leak comes from an external dependency or shall be handled later then it is possible to set a function on a suppression list.
+
+If errors occur, an error report is shown with a stack trace to find the place where the leak occurs. If the leak comes from an external dependency or shall be handled later then it is possible to set a function on a suppression list.
 This should be only rarely used and only in coordination with an iceoryx maintainer.
 
-**NOTE**
-Iceoryx needs to be build as static library for working with sanitizer flags. The script does it automatically.
+!!! note
+    iceoryx needs to be built as a static library for working with sanitizer flags. The script does it automatically.
+    Except when you want to use the ${ICEORYX_WARNINGS} then you have to call `findpackage(iceoryx_utils)`
 
 ## Iceoryx library build
```

Comment:
iceoryx should be *always* spelled with a lower capital. We can fix this in a follow-up